packages/unchained-client/src/evm/ethereum/parser/constants.ts

```diff
@@ -7,3 +7,4 @@ export const FOXY_STAKING_CONTRACT = '0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b
 export const WETH_CONTRACT_MAINNET = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
 export const WETH_CONTRACT_ROPSTEN = '0xc778417E063141139Fce010982780140Aa0cD5Ab'
 export const UNI_V2_FOX_STAKING_REWARDS_V3 = '0x212ebf9FD3c10F371557b08E993eAaB385c3932b'
+export const COWSWAP_CONTRACT_MAINNET = '0x9008D19f58AAbD9eD0D60971565AA8510560ab41'
```

Comment:
reference tx for the astute reviewer

https://etherscan.io/tx/0x6796daaa190af29881b019b8d1119543086edf96c194edaf7d49dea8d100faa9